Gemfile

```diff
@@ -1,4 +1,3 @@
 source "https://rubygems.org"
 
-gem "minitest"
-gem "debug", ">= 1.0.0"
\ No newline at end of file
+gem 'minitest', '~> 5.16'
```

Comment:
Los Gemfiles también pueden declarar dependencias de desarrollo; esto tenía que haberlo dicho en el aterior, pero ya que lo has modificado, te lo digo aquí. Mira por ejemplo [esto](https://medium.com/@dvandersluis/using-development-dependencies-from-git-in-your-gem-1736090db1d0)

Reply:
De acuerdo, me ha quedado claro esto, ya no lo modifico pero lo tengo en cuenta de cara al futuro.

Reply:
Si puedes, hazlo en este objetivo.